operations/agent-flow-mixin/dashboards/prometheus.libsonnet

```diff
@@ -19,7 +19,7 @@ local scrapePanels(y_offset) = [
 
   // Scrape success rate
   (
-    panel.new(title='Scrape success rate', type='timeseries') +
+    panel.new(title='Scrape success rate in $cluster', type='timeseries') +
     panel.withUnit('percent') +
```

Comment:
Would it make more sense to change the unit than the query?

Reply:
Yeah, I think it's `percentunit` for a scale of 0-1

Reply:
TIL :D thanks!